JWBestPracticeApps/JWBestPracticeApps.xcodeproj/project.pbxproj

```diff
@@ -1727,6 +1727,7 @@
 				TargetAttributes = {
 					3B181AD223592D5A003D4CDB = {
 						CreatedOnToolsVersion = 11.1;
+						DevelopmentTeam = WX87M56X5N;
```

Comment:
@MikeSalvador we probably shouldn't commit these since our uses will have their own credentials. They will run into issues when building and might not know how to address if they are junior